server/migrations/init.sql

```diff
@@ -11,24 +11,24 @@ CREATE EXTENSION citext;
 CREATE TABLE IF NOT EXISTS rentr_user (
     id SERIAL PRIMARY KEY,
     email CITEXT UNIQUE NOT NULL,
-    password VARCHAR(500) NOT NULL
+    password VARCHAR CHECK (length(password) <= 100) NOT NULL
```

Comment:
I'm guessing you tested that the hashes are less than 100 chars so we don't need it to be 500? Looks good otherwise!

Reply:
Yeah. The encrypted password is 60 characters. Extra 40 characters are just in case the package updates itself and changes the output length after the encryption.